kani-driver/src/call_cbmc.rs

```diff
@@ -40,7 +41,9 @@ impl KaniSession {
             // extra argument
             cmd.arg("--json-ui");
 
+            let now = Instant::now();
             let _cbmc_result = self.run_redirect(cmd, &output_filename)?;
+            let elapsed = now.elapsed().as_secs_f32();
```

Comment:
Are there any possible scenarios where there might be no time captured in the variable elapsed?